### DISCUSSION TITLE
Desktop Display for BirdNet-Pi

### DISCUSSION THREAD

**Discussion by ThomDyson** [Nachtzuster/BirdNET-Pi#231] [Show and tell]
BirdNet-Pi is great, but I didn't want to have to keep pulling my phone out to see what's up. I build a desktop display from anESP32 powered "Cheap Yellow Display". The CYD is a $20 2.8" touchscreen LCD. I show the last 10 birds, with confidence level and time of last notification. I have relatives who would love a BirdNet-Pi, but would never look at the web pages, so this will be a good Christmas gift.

![sample display screen](https://github.com/user-attachments/assets/fab9db29-e36d-4e16-a32f-85d3786b2273)

[BirdNet Display](https://github.com/ThomDyson/BirdNetDisplay)

**Comment by rmcmullanLB** [comment]
Thanks for sharing! I'm going to print out a case for the CYD and build one for my wife.
